pkg/backend/benchmark_test.go

```diff
@@ -1,350 +0,0 @@
-package backend
```

Comment:
Why this test is deleted? Looks useful to me

Reply:
I don't think this measurement is in any way relevant at the moment. Tests like this introduce dependencies in the code and slow down the work while not bringing value.

The plan is to remove the majority of unit-tests for the backends as well as *all* http unit tests for zipper.